config/core/configmaps/features.yaml

```diff
@@ -22,3 +22,6 @@ metadata:
     knative.dev/config-propagation: original
     knative.dev/config-category: eventing
 data:
+  # ALPHA feature: The kreference-group allows you to use the Group field in KReferences.
+  # For more details: https://github.com/knative/eventing/issues/5086
+  kreference-group: "disabled"
```

Comment:
It's probably fine for now but eventually it should look more like https://github.com/knative/serving/blob/main/config/core/configmaps/features.yaml.

Reply:
Mhh yeah seems like we don't have the example, but do we really need it? Isn't simpler for the user to have the flags already configured? Also, considering at some point some of them will become enabled by default?